celery/concurrency/eventlet.py

```diff
@@ -107,8 +109,9 @@ def __init__(self, *args, **kwargs):
 
     def on_start(self):
         self._pool = self.Pool(self.limit)
+        self._pool_map = {}
         signals.eventlet_pool_started.send(sender=self)
-        self._quick_put = self._pool.spawn_n
+        self._quick_put = self._pool.spawn
```

Comment:
Why did we change `spawn_n` to `spawn`?

Reply:
According to [documentation](http://eventlet.net/doc/modules/greenpool.html#eventlet.greenpool.GreenPool.spawn_n) `spawn_n` returns None while `spawn` returns `greenlet`/`greenthread`.